tests/tck/features/lookup/LookupTag2.feature

```diff
@@ -28,6 +28,11 @@ Feature: Test lookup on tag index 2
       """
 
   Scenario Outline: [tag] simple tag test cases
+    When executing query:
+      """
+      LOOKUP ON lookup_tag_1 WHERE lookup_tag_1.col1 == 201 OR lookup_tag_1.col2 == 201 AND lookup_tag_1.col3 == 202
+      """
+    Then the execution should be successful
```

Comment:
Check result

Reply:
This test intends to test the semantics.
Tests with result checking are in https://github.com/vesoft-inc/nebula/pull/2906/files/f23b916ab47e691e6049271c0f4b9b4a0e6b7051#diff-593c691acd0611750cea4018240e111363ce5b7e074f9a6a378ce877ea8574f7R109-R123